### DISCUSSION TITLE
[Unstable] Alarm sound is doubled

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#14242] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

alarm sounds for empty diving suits, and reactor meltdown seem to be doubled

### Reproduction steps

put on a suit with no oxygen, make your reactor meltdown

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14230
